tutorials/desktop/ubuntu-on-windows/ubuntu-on-windows.md

```diff
@@ -0,0 +1,124 @@
+---
+id: tutorial-ubuntu-on-windows
+summary: Get access to the unrivalled power of the Ubuntu terminal, including tools such as SSH, apt and vim, directly on your Windows 10 computer.
+categories: desktop
+tags: tutorial,installation,windows,ubuntu,terminal
+difficulty: 2
+status: Published
+published: 2017-08-29
+author: Graham Morrison <graham.morrison@canonical.com>
+
+---
+
+# Install Ubuntu on Windows 10
+Duration: 1:00
+
+## Overview
+
+The wonderful Ubuntu terminal is [freely available][msubuntu] for Windows 10.
+
+As any Linux user knows, it's the command line terminal where the magic happens. It's perfect for file management, development, remote administration and a thousand other tasks.
+
+The Ubuntu terminal for Windows has many of the same features you'll find using the terminal on Ubuntu:
+
+- Unrivalled breadth of packages, updates and security features
+- Bash, Z-Shell, Korn and other shell environments without virtual machines or dual-booting
+- Run native tools such as SSH, git, apt and dpkg directly from your Windows computer
+- A huge community of friendly, approachable users
+
+positive
+: Currently, packages that require a graphical interface will not work from the Ubuntu terminal.
```

Comment:
Since we might mostly have an audience of developers that are switching from Windows based dev to Linux dev, "applications" could be clearer for them.

Reply:
good point, thanks!